### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 25

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#351] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325863-0bd43432-7340-467f-af54-dc08d18130d3.png)

**Comment by LeuxLello** [comment]
![20220614_142754](https://user-images.githubusercontent.com/104726379/173577403-dc584626-597d-4e25-b557-e7bcbb2eb9ef.jpg)
![20220614_142747](https://user-images.githubusercontent.com/104726379/173577419-804fc4d7-4a8b-4b04-8ddd-9fa847dc4805.jpg)

**Comment by LeuxLello** [comment]
dovrebbero essere giusti i procedimenti ma non sono sicuro

**Comment by LuginiAndrea** [comment]
1. Sat
2. UnSat
3. UnSat
